app/mailers/referee_mailer.rb

```diff
@@ -86,4 +101,15 @@ def google_form_url_for(candidate_name, reference)
       }.to_query.gsub('+', '%20')
   end
   # rubocop:enable  Style/StringConcatenation
+
+  def decide_guidance_by_referee_type(reference)
+    case reference.referee_type.to_sym
+    when :academic
+      'their academic performance'
+    when :school_based, :professional
+      'their role and responsibilities at work'
+    else
+      'activities you’ve done together'
+    end
+  end
```

Comment:
Feels odd to me to have this condition in the mailer class. Could move to a locale file and pass in the reference type?

Something like this maybe in the mailer view:

```erb
  <%= t(:referee_mail, :guidance, @referee_type %>
```

and then in the locale file:


```yaml
referee_mail:
  guidance:
    academic: their academic performance
    school_based: their role and responsibilities at work
    professional: their role and responsibilities at work
    character: activities you’ve done together
```

Just an idea though!

Reply:
Either works fine I think, but I prefer this, added 👍